poppy/wfe.py

```diff
@@ -353,6 +353,231 @@ def get_opd(self, wave):
         phase_screen = np.fft.ifftshift(np.fft.ifft2(np.fft.ifftshift(scaled))).real   # FT of scaled random PSD makes phase screen
 
         phase_screen -= np.mean(phase_screen)  # force zero-mean
-        opd = phase_screen / np.std(phase_screen) * self.wfe.to(u.m).value  # normalize to wanted input rms wfe
+        self.opd = phase_screen / np.std(phase_screen) * self.wfe.to(u.m).value  # normalize to wanted input rms wfe
 
-        return opd
+        return self.opd
+
+
+class PowerSpectrumWFE(WavefrontError):
+    """
+    WFE model specificed via a Power Spectral Density (PSD),
+    or a list of multiple PSDs, which follow von Karman PSD model:
+    
+    :math: `P(k) = \frac{\beta} {\left( \left(\frac{1}{L_{0}}\right)^{2} + |k|^{2} \right)^{{\alpha/2}}} e^{-(|k|l_{0})^{2}} + \beta_{sr}`
```

Comment:
You’ll want to remove the space between the directive and the text enclosed in backticks for it to render properly. Since backticks are used for formatting I couldn’t give an explicit example in Slack from my phone 😛here’s a better one

```
:math:`x^2`
```

Reply:
Okay, thanks for the correction. Does this continue to hold if I wanted to have a string next to the LaTeX variable? For example: `:math:`L_{0}`: astropy quantity`

Reply:
Yep backticks are overloaded

Reply:
The parser sees it as

```
:<name>:`<content>`
```

So anything after the second backtick shouldn’t matter. You could put a colon right there, or a comma, or another colon thing (i think docutils calls them “roles”?)